.changeset/nine-paws-unite.md

```diff
@@ -0,0 +1,12 @@
+---
+'sku': major
+---
+
+`sku init` no longer installs and configures husky for you
+
+BREAKING CHANGE
+
+`sku init` no longer adds `husky` as a dependecy nor does it configure husky for you out of the box.
+For instructions on how to set up `husky` to use sku's pre-commit hook, see [the docs].
+
+[the docs]: https://seek-oss.github.io/sku/#/./docs/extra-features?id=pre-commit-hooks
```

Comment:
You changed the heading so this must change as well.

```suggestion
[the docs]: https://seek-oss.github.io/sku/#/./docs/extra-features?id=pre-commit-hook
```

Reply:
Good catch 👍